extension/backend.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/ethereum/go-ethereum/params"
```

Comment:
should be grouped below in the github.com imports, goimports linting is failing

Reply:
done